File: ShapeByte/features/workout/schedule/data/WorkoutScheduleEntry.swift
//
//  File.swift
//  ShapeByte
//
//  Created by Lang, Stefan [RTL Tech] on 02.08.24.
//

import Foundation

struct WorkoutScheduleEntry: Equatable {
    let id: String
    let name: String
    let date: Date
    let progress: Progress
}

File: features/workout/data/contract/src/commonMain/kotlin/de/stefan/lang/shapebyte/features/workout/data/contract/schedule/WorkoutScheduleEntry.kt
package de.stefan.lang.shapebyte.features.workout.data.contract.schedule

import de.stefan.lang.coreutils.contract.progress.Progress
import kotlinx.datetime.Instant

public data class WorkoutScheduleEntry public constructor(
    public val id: String,
    public val name: String,
    public val date: Instant,
    public val progress: Progress,
)
